flink/README.md

```diff
@@ -53,17 +53,9 @@ No additional installation is needed on your server.
 
 _Available for Agent >6.0_
 
-1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j.properties`, `log4j-cli.properties`, `log4j-yarn-session.properties`, or `log4j-console.properties` file. See [Flink's repository][6] for default configurations. For example `log4j.properties` contains this configuration by default:
+1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j*.properties` configuration files in the `conf/` directory of the Flink distribution. See the [Flink logging documentation][13] for which configuration file is relevant for your setup. See [Flink's repository][6] for default configurations.
```

Comment:
```suggestion
1. Flink uses the `log4j` logger by default. To enable logging to a file, customize the format by editing the `log4j*.properties` configuration files in the `conf/` directory of the Flink distribution. See the [Flink logging documentation][13] for information on which configuration file is relevant for your setup. See [Flink's repository][6] for default configurations.
```

Reply:
I could be very wrong with my interpretation of the above step. Please let me know if that's the case.